desktop/cmp/grid/find/GridFindField.js

```diff
@@ -61,33 +61,47 @@ export const [GridFindField, gridFindField] = hoistCmp.withFactory({
                     leftIcon: Icon.search(),
                     enableClear: true,
                     placeholder: 'Find',
+                    selectOnFocus: true,
                     width: null,
                     flex: 1,
-                    onKeyDown: (ev) => {
-                        if (ev.key === 'Enter') {
-                            ev.shiftKey ? impl.selectPrev() : impl.selectNext();
+                    onKeyDown: (e) => {
+                        switch (e.key) {
+                            case 'Enter':
+                                e.shiftKey ? impl.selectPrev() : impl.selectNext();
+                                consumeEvent(e);
+                                break;
+                            case 'ArrowUp':
+                                impl.selectPrev();
+                                consumeEvent(e);
+                                break;
+                            case 'ArrowDown':
+                                impl.selectNext();
+                                consumeEvent(e);
+                                break;
                         }
                     },
                     ...restProps
                 }),
                 hbox({
+                    omit: !countLabel,
```

Comment:
Unrelated to this change, but I wonder if we can show / hide the controls based on focus rather than results? I feel like that might be the most natural time to reveal them